Userland/Games/BrickGame/BrickGame.cpp

```diff
@@ -508,6 +548,30 @@ void BrickGame::paint_text(GUI::Painter& painter, int row, DeprecatedString cons
     painter.draw_text(rect, text, Gfx::TextAlignment::TopLeft, Color::Black);
 }
 
+void BrickGame::paint_paused_text(GUI::Painter& painter)
+{
+    auto const paused_text = "Paused"sv;
+    auto const paused_text_width = static_cast<int>(ceilf(font().width(paused_text)));
+    auto const more_or_less_font_height = static_cast<int>(font().pixel_size_rounded_up());
+    auto const entire_area_rect { frame_inner_rect() };
+    auto const margin = more_or_less_font_height * 2;
+
+    Gfx::IntRect pause_text_box {
+        entire_area_rect.x() + entire_area_rect.width() / 2,
+        entire_area_rect.y() + entire_area_rect.height() / 2,
+        0,
+        0
+    };
+
+    pause_text_box.inflate(paused_text_width + margin, more_or_less_font_height + margin);
```

Comment:
Can this just use `centered_within()`?
```suggestion
   auto pause_text_box = Gfx::IntRect({}, { paused_text_width + margin, more_or_less_font_height + margin }).centered_within(entire_area_rect);
```

Reply:
>Can this just use `centered_within()`?

It can, thank you! Changed.